### DISCUSSION TITLE
Error when installing with Vim Plug

### DISCUSSION THREAD

**Discussion by chinleung** [pwntester/octo.nvim#359] [Q&A] [answered]
Hi,

When I'm trying to install the plugin with:

```
function! SetupOcto(info)
    require "octo".setup();
endfunction

Plug 'pwntester/octo.nvim', { 'do': function('SetupOcto') }
```

I'm getting the following error:

> Not an editor command: require "octo".setup();

**Comment by brendalf** [comment]
The same issue happens if you write as: `require("octo").setup()`?

**Reply by chinleung** [reply]
> The same issue happens if you write as: `require("octo").setup()`?

I've tested it and I get the same error.

**Comment by chinleung** [comment, marked as answer]
I had to use the following to make it work:

```
call plug#begin()
    ...
    Plug 'pwntester/octo.nvim'
call plug#end()

lua require('octo').setup()
```

**Reply by pwntester** [reply]
Sorry for the late response. Yep, seems like you were using vimscript and then calling a lua function. You need to use something like

```
function! SetupOcto(info)
lua << EOF
require "octo".setup();
EOF  
endfunction

Plug 'pwntester/octo.nvim', { 'do': function('SetupOcto') }
```

or as you ended up doing `lua require('octo').setup()`
